recipes/google-cloud-storage/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.1.1" %}
+
+package:
+  name: google-cloud-storage
+  version: {{ version }}
+
+source:
+  fn: google-cloud-storage-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/g/google-cloud-storage/google-cloud-storage-{{ version }}.tar.gz
+  sha256: 11e709ed37523eb1bb68fed4dd31fb176cd7734be89cbfc5c22f1898562041bd
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - requests >=2.0.0
+
+  run:
+    - python
+    - google-cloud-core >=0.24.1,<0.25dev
+    - google-auth >=1.0.0
+    - google-resumable-media ==0.0.2
+    - requests >=2.0.0
+
+test:
+  imports:
+    - google
+    - google.cloud
+    - google.cloud.storage
+
+about:
+  home: https://github.com/GoogleCloudPlatform/google-cloud-python
+  summary: Python Client for Google Cloud Storage
+  doc_url: https://googlecloudplatform.github.io/google-cloud-python/stable/google-cloud-api.html
+  dev_url: https://github.com/GoogleCloudPlatform/google-cloud-python/tree/master/storage
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: {{ RECIPE_DIR }}/LICENSE
```

Comment:
Looking on pypi.io, it looks like the `LICENSE` file is packaged with the source, so I don't think you need to add it manually. You can download the source and double check that.